src/gui_node.rs

```diff
@@ -26,23 +25,21 @@ pub struct GuiNodeData {
 
 pub type GuiNodeDataRef = Rc<RefCell<GuiNodeData>>;
 
+#[derive(Clone, WidgetCommon)]
 pub struct GuiNode {
-    common: widget::CommonBuilder,
+    #[conrod(common_builder)] common: widget::CommonBuilder,
     data: GuiNodeDataRef,
     style: Style,
     selected: bool,
 }
 
-
 // We use the `widget_style!` macro to vastly simplify the definition and implementation of the
 // widget's associated `Style` type. This generates both a `Style` struct, as well as an
 // implementation that automatically retrieves defaults from the provided theme.
 //
 // See the documenation of the macro for a more details.
-widget_style! {
-    style Style {
-    }
-}
+#[derive(Copy, Clone, Debug, Default, PartialEq, WidgetStyle)]
```

Comment:
There's no more widget_style! macro; you derive WidgetStyle (hence the need for the conrod_derive procedural macro crate).